src/Microsoft.ComponentDetection.Detectors/nuget/NuGetPackageReferenceFrameworkAwareDetector.cs

```diff
@@ -192,13 +192,12 @@ private void RegisterPackageDownloads(ISingleFileComponentRecorder singleFileCom
 
                 var libraryComponent = new DetectedComponent(new NuGetComponent(packageDownload.Name, packageDownload.VersionRange.MinVersion.ToNormalizedString()));
 
-                // Conservatively assume that PackageDownloads are not develeopment dependencies even though usage will not effect any runtime behavior.
-                // Most often they are used for some runtime deployment -- runtime packs, host packs, AOT infrastructure, etc, so opt in treating them as non-development-dependencies.
+                // PackageDownload is always a development dependency since it's usage does not make it part of the application
                 singleFileComponentRecorder.RegisterUsage(
                     libraryComponent,
                     isExplicitReferencedDependency: true,
                     parentComponentId: null,
-                    isDevelopmentDependency: false,
+                    isDevelopmentDependency: true,
```

Comment:
Do we have any tests for this? I was expecting something to break with this type of change, we should probably at one that validates PackageDownloads always go as dev dependencies.

Reply:
We test that they appear but not whether they should be devDependencies or not.  I've added a test.